qa/automation/Starynina.Scenarios/3. Send CV/3.2.Send CV.feature

```diff
@@ -0,0 +1,136 @@
+Feature: 3.2. Send CV
+
+Background:
+Given user is on the page with description of QA Engineer (Life Science project) position https://career.quantori.com/ru/positions/qa-engineer-life-science-project
+
+Scenario: 3.2_01 User can use the "Send CV" button
+	When user clicks the "Send CV" button
+	Then the page should be scrolled down to the CV form
+
+Scenario: 3.2_02 User can scroll the page
+	When user uses the scroll on the side bar
+	Then the page should be scrolled down to the CV form
+
+@negative
+Scenario: 3.2_03 User can't send empty CV form
+	When user clicks the "Send" button
+	Then the error messages "This field is required" for "Name" and "Email" fields is appeared
+	And the "Name" and "Email" field, "Attach CV" button and the checkbox "I agree to the personal data processing" are highligted in red
+	And the CV form should not be sent
+
+Scenario: 3.2_04 User can choose CV file from local PC
+	When user clicks the "Attach CV" button
+	Then the window to choose the file from local PC should be opened
+
+Scenario Outline: 3.2_05 User enters various types of name in the CV form 
```

Comment:
You should separate the Scenario for the Negative and Positive results.

Reply:
Corrected